deps/devdoc/requirement_docs/com/microsoft/azure/sdk/iot/deps/serializer/authenticationparser_requirements.md

```diff
@@ -1,73 +0,0 @@
-# AuthenticationParser Requirements
-
-## Overview
-
-Representation of an Authentication object used for Json deserialization and serialization.
-
-## References
-
-
-## Exposed API
-
-```java
-public class AuthenticationParser
-{
-    public SymmetricKeyParser getSymmetricKey()
-
-    public void setSymmetricKey(SymmetricKeyParser symmetricKey)
-    public X509ThumbprintParser getThumbprint()
-    public void setThumbprint(X509ThumbprintParser thumbprint)
-    public AuthenticationTypeParser getType()
-    public void setType(AuthenticationTypeParser type) throws IllegalArgumentException
-}
-```
-
-
-### AuthenticationParser
-```java
-protected AuthenticationParser();
-```
-**SRS_AUTHENTICATION_PARSER_34_001: [**This Constructor shall create a new instance of an authenticationParser object and return it.**]**
-
-
-### getType
-```java
-public AuthenticationTypeParser getType()
-```
-**SRS_AUTHENTICATION_PARSER_34_002: [**This method shall return the value of this object's authenticationTypeParser.**]**
-
-
-### setType
-```java
-public void setType(AuthenticationTypeParser type)
-````
-**SRS_AUTHENTICATION_PARSER_34_008: [**This method shall set the value of this object's authentication type equal to the provided value.**]**
```

Comment:
But the references of these still exist in source code right?

Reply:
They do, but I've been deleting them slowly over time as they go out of date